app/assets/javascripts/test/tracing-view-screenshot-tests/dataset_rendering.screenshot.js

```diff
@@ -0,0 +1,124 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
```

Comment:
I'd rename the folder "tracing-view-screenshot-tests" to "puppeteer", since theoretically we could use puppeteer for other tests, as well.